hubblestack/extmods/modules/pulsar.py

```diff
@@ -273,6 +300,8 @@ def process(configfile='salt://hubblestack_pulsar/hubblestack_pulsar_config.yaml
             else:
                 log.info('Excluding {0} from event for {1}'.format(event.pathname, path))
 
+        print("-------------------------%s time------------------------" % (time.time() - start_time))
```

Comment:
This seems to be leftover from your debugging, but I don't think we want to print in the final version.